url.bs

```diff
@@ -849,21 +1180,22 @@ return value of the <a for=/>host parser</a> is an <a for=/>IPv4 address</a>.
    <li><p>Let <var>result</var> be the result of <a lt="IPv4 number parser">parsing</a>
    <var>part</var>.
 
-   <li><p>If <var>result</var> is failure, <a>validation error</a>, return failure.
+   <li><p>If <var>result</var> is failure, <a>non-numeric-IPv4-part</a> <a>validation error</a>,
+   return failure.
 
-   <li><p>If <var>result</var>[1] is true, <a>validation error</a>.
+   <li><p>If <var>result</var>[1] is true, <a>non-decimal-IPv4-part</a> <a>validation error</a>.
 
    <li><p><a for=list>Append</a> <var>result</var>[0] to <var>numbers</var>.
   </ol>
 
- <li><p>If any item in <var>numbers</var> is greater than 255, <a>validation error</a>.
+ <li><p>If any item in <var>numbers</var> is greater than 255, <a>IPv4-part-out-of-range</a>
+ <a>validation error</a>.
 
  <li><p>If any but the last <a for=list>item</a> in <var>numbers</var> is greater than 255, then
  return failure.
 
  <li><p>If the last <a for=list>item</a> in <var>numbers</var> is greater than or equal to
- 256<sup>(5 &minus; <var>numbers</var>'s <a for=list>size</a>)</sup>, <a>validation error</a>,
```

Comment:
Is the validation error that was removed here always caught by Step 6 above, where it checks for any number > 255?

Reply:
Yeah, numbers cannot contain more than 4 items.